src/bin/huak/main.rs

```diff
@@ -1,14 +1,14 @@
 //! The `huak` application.
 //!
 //! Huak implements a cli application with various subcommands.
-use clap::{self, ArgMatches};
+use clap::{self, AppSettings, ArgMatches};
 use huak::errors::{CliError, CliResult};
 
 mod commands;
 
 /// Launch Huak's cli process.
 pub fn main() -> CliResult {
-    let args = commands::args();
+    let args = commands::args().setting(AppSettings::ArgRequiredElseHelp);
```

Comment:
```suggestion
    let args = commands::args()
        .version(clap::crate_version!())
        .author(clap::crate_authors!())
        .about("A Python package manager written in Rust inspired by Cargo")
        .setting(AppSettings::ArgRequiredElseHelp);
```

I think this is doable. Then we'd just remove it from commands::mod. This would be a welcome change as part of this PR.

Reply:
1 sec

Reply:
looks good to me

Reply:
Just a heads up this will resolve and pass ci-rust but https://github.com/dpgraham4401/huak/blob/290d13556226375ce202b730eb09e4c04ed08c4a/src/bin/huak/commands/mod.rs#L25-L27 would just need to be removed before we can merge.

Reply:
I'll get it for you.